add-scene-layer-with-elevation/src/main/java/com/esri/arcgismaps/sample/addscenelayerwithelevation/screens/MainScreen.kt

```diff
@@ -44,10 +44,9 @@ fun MainScreen(sampleName: String, application: Application) {
                     .padding(it)
```

Comment:
```
// get the application context
val application = LocalContext.current.applicationContext as Application
```
Instead of passing the application from MainActivity, get it using local context.